### PR TITLE
✨#631 add notification body

### PR DESCRIPTION
**Pull request by Macludde**
When creating an article, and choose to send notifications to tag subscribers, a new input field is presented where the text of the notification.

The header of the article is always present, but the body of the notification will be what's in this input field. If left empty, no body will be present on the notification. In other words, the regular body of the article is never used in the notification.

### CODE REVIEW

**Comment by oliverlevay** [comment]
Fix the lint errors! @Macludde is eslint not working for you?

**Comment by Macludde** [comment]
> Fix the lint errors! @Macludde is eslint not working for you?

Hmmm... no I did not get anything on my end. My dev env is pretty screwed up however, seems like eslint can't find tsconfig (see image)


image

**Comment by oliverlevay** [comment]
> > Fix the lint errors! @Macludde is eslint not working for you?
> 
> Hmmm... no I did not get anything on my end. My dev env is pretty screwed up however, seems like eslint can't find tsconfig (see image)
> 
> image

Weird! I wonder why it's looking in `member-page/frontend/backend`

**Comment by Macludde** [comment]
> > > Fix the lint errors! @Macludde is eslint not working for you?
> > 
> > 
> > Hmmm... no I did not get anything on my end. My dev env is pretty screwed up however, seems like eslint can't find tsconfig (see image)
> > image
> 
> Weird! I wonder why it's looking in `member-page/frontend/backend`

Managed to fix it, was an issue with our monorepo and me using "code workspaces" in vscode. If I only open the member page folder it works.

**Comment by oliverlevay** [comment]
I'll review the code when the tests pass : )

File: backend/services/core/src/datasources/News.ts [hunk 1]
```diff
@@ -479,7 +480,7 @@ export default class News extends dbUtils.KnexDataSource {
     });
   }
 
-  private async sendNotifications(title: string, body: string, tagIds?: UUID[], data?: Object) {
+  private async sendNotifications(title?: string, body?: string, tagIds?: UUID[], data?: Object) {
```
Comment (oliverlevay) [inline backend/services/core/src/datasources/News.ts:483]: Title should be required? 🤔

File: frontend/components/ArticleEditor/ArticleEditorItem/index.tsx [hunk 2]
```diff
@@ -1,10 +1,9 @@
-import React from 'react';
 import { useTranslation } from 'next-i18next';
+import React from 'react';
 // @ts-ignore package does not have typescript types
```
Comment (oliverlevay) [inline frontend/components/ArticleEditor/ArticleEditorItem/index.tsx:3]: Hmm strange, this comment can probably be removed?

**Review by oliverlevay — APPROVED** [review]
🥇 Superb

nothing that needs to be fixed, only two things that confuse me, LGTM

**Pull request merged**